packages/server/graphql/private/mutations/processRecurrence.ts

```diff
@@ -81,6 +81,11 @@ const processRecurrence: MutationResolvers['processRecurrence'] = async (_source
   const activeMeetingSeries = await getActiveMeetingSeries()
   await Promise.all(
     activeMeetingSeries.map(async (meetingSeries) => {
+      const seriesTeam = await dataLoader.get('teams').loadNonNull(meetingSeries.teamId)
+      if (seriesTeam.isArchived) {
```

Comment:
good catch! now, should we also check if organization is locked?
CC @nickoferrall

Reply:
Good question! Probably not, but curious what @nickoferrall thinks.

I just pushed a commit that does this, but I can revert if that's not what we want.

Reply:
As we're preventing other meetings from starting if it's locked, I'd say we should also prevent standups from recurring if the org is locked or the team's `isPaid` field is false. See logic [here](https://github.com/ParabolInc/parabol/blob/b99805925134225e941241fa459cdf0d68b2ab49/packages/server/graphql/mutations/helpers/isStartMeetingLocked.ts)